lib/Scraper/Scraper.php

```diff
@@ -37,7 +37,7 @@ public function __construct(LoggerInterface $logger)
             CURLOPT_RETURNTRANSFER => true,     // return web page
             CURLOPT_HEADER         => false,    // do not return headers
             CURLOPT_FOLLOWLOCATION => true,     // follow redirects
-            //CURLOPT_USERAGENT    => "php-news", // who am i
+            CURLOPT_USERAGENT      => "Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/117.0.0.0 Safari/537.36", // who am i
```

Comment:
It should either use the defined news user agent or none at all

Reply:
Now, as far as I understand, there is no way to set a custom user agent.
Do you propose to add this point to the news settings?
If you don't use a user agent at all, some sites won't serve content.

Reply:
There is https://github.com/nextcloud/news/blob/953731604e2489ab63579865a248093c405be96e/lib/Config/FetcherConfig.php#L51 that is the default news useragent. It should use that.

Reply:
> It should use that.

I changed the code.